uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/APIWrapper/DefaultLimitRemoverTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest
+ *
+ * @group headless
+ */
+class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  private $defaultLimitRemoverObj;
+
+  private $testParameters = [];
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->installMe(__DIR__)
+      ->apply();
+  }
+
+  public function setUp() {
+    $this->defaultLimitRemoverObj = new CRM_HRCore_APIWrapper_DefaultLimitRemover();
+
+    $this->testParameters['params'] = [
+      'action' => 'get',
+      'entity' => 'contact',
+      'id' => 1,
+    ];
```

Comment:
Things like action, entity and id are not relevant to most of the tests, so there's no reason to have this on all of them. I think you can remove this variable altogether